benchmarks/ekomark/plots.py

```diff
@@ -193,6 +193,8 @@ def plot_operator(var_name, op, op_err, log_operator=True, abs_operator=True):
 
     ax = plt.subplot(1, 3, 3)
     plt.title("|error/value|")
+    # TODO: next line will cause a "invalid value encountered in true divide
+    np.seterr(divide="ignore", invalid="ignore")
```

Comment:
you can do that, but you should revert afterwards: https://numpy.org/doc/stable/reference/generated/numpy.seterr.html?highlight=seterr#numpy.seterr